tests/integration/security/authorization_test.go

```diff
@@ -1363,5 +1368,36 @@ func TestAuthorization_Custom(t *testing.T) {
 			}
 
 			rbacUtil.RunRBACTest(ctx, cases)
+
+			type ingressCase struct {
+				Name     string
+				Host     string
+				Path     string
+				IP       string
+				WantCode int
+			}
+			newIngressCase := func(Name string, Host string, Path string, IP string, WantCode int) ingressCase {
+				return ingressCase{
+					Name:     Name,
+					Host:     Host,
+					Path:     Path,
+					IP:       IP,
+					WantCode: WantCode,
+				}
+			}
+
+			ingressCases := []ingressCase{
+				newIngressCase("deny www.company.com", "www.company.com", "/", "172.16.0.1", 403),
+				newIngressCase("allow www.company.com", "www.company.com", "/custom", "172.16.0.1", 200),
+			}
+			for _, tc := range ingressCases {
+				ctx.NewSubTest(tc.Name).Run(func(ctx framework.TestContext) {
+					headers := map[string][]string{
+						"X-Forwarded-For": {tc.IP},
```

Comment:
No need to set the `X-Forwarded-For`, also set `x-ext-authz` like other test cases (some use allow and some use deny).